__tests__/components/Comments/WriteComment.test.js

```diff
@@ -49,16 +40,7 @@ describe('WriteComment should run correctly', () => {
       fontSize: 12,
       color: COLORS.blackgrey
     });
-    expect(styles.container(true)).toEqual({
-      flex: 1,
-      backgroundColor: COLORS.white,
-      width: '100%',
-      paddingRight: 20,
-      paddingLeft: 43,
-      flexDirection: 'row',
-      zIndex: 100
-    });
-    expect(styles.container(false)).toEqual({
+    expect(styles.container).toEqual({
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the new code has removed the conditionality of the `styles.container` style. Please ensure that all instances where this style is used have been updated accordingly to avoid potential issues.
```diff
-    expect(styles.container).toEqual({
+    expect(styles.container(true)).toEqual({
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->